sdk/cpp/tests/test_netconf_service.cpp

```diff
@@ -328,30 +328,3 @@ TEST_CASE("ietf_get_rpc")
         cout << "Exception while executing RPC: " << ex.what() << endl;
     }
 }
-
-TEST_CASE("get_config_openconfig_interfaces_and_bgp")
```

Comment:
I see. It's probably better to comment it out with that explanation rather than deleting everything.

See this example:
https://github.com/ygorelik/ydk-gen/blob/715bed51b22bf3cb9e1e8d69aee23f9ee2d9544e/sdk/cpp/tests/test_netconf_service.cpp#L40